addons/arsenal/functions/fnc_buttonExport.sqf

```diff
@@ -16,7 +16,7 @@ params ["_display"];
 
 if (GVAR(shiftState)) then {
 
-    if (isNil QGVAR(defaultLoadoutsList) || {GVAR(defaultLoadoutsList) isEqualTo []}) exitWith {
```

Comment:
This is unscheduled anyway so nil shouldn't matter right?

Reply:
it can't be nil, so no worries. (my mistake)